internal/outputer/formatter/policies_content.go

```diff
@@ -34,6 +34,24 @@ func newPoliciesContent(pf policiesFormatter, colorizer colorizer) *policiesCont
 	}
 }
 
+func (pc *policiesContent) FormatPolicy(output *scheme.Flattened, policyName string) []byte {
+	pc.sb.Reset()
+
+	if _, ok := output.AsOrderedMap().Get(policyName); ok {
+		data := output.GetPolicyData(policyName)
+
+		pc.writeLine(pc.pf.FormatTitle(data.PolicyInfo.Title, data.PolicyInfo.Severity))
+
+		pc.depth++
+		pc.writePolicyInfo(policyName, data.PolicyInfo)
+		pc.writeLineBreak()
+		pc.writeViolations(data.Violations)
+		pc.depth--
```

Comment:
looks like all this code is a duplicate of the code from FormatFailedPolicies().
it'd be nice if we move this code to an helper (internal) function, e.g. `formatPolicyInfo(policyName)`,
and call that in both.